mscore/CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ QT5_WRAP_UI (ui_headers
       synthcontrol.ui splitstaff.ui selectinstr.ui
       editstafftype.ui
       editpitch.ui editstringdata.ui editraster.ui mediadialog.ui albummanager.ui layer.ui realizeharmonydialog.ui
-      omrpanel.ui masterpalette.ui harmonyedit.ui pathlistdialog.ui
```

Comment:
I'm not quite sure how it relates to your PR?

Reply:
Someone added the `masterpalette.ui` file in this CMakeList in the master, but it is not in the `mscore`, it is in `mu4/scenes/palette`